tests/ui-testing/pages/ingestionPage.js

```
@@ -0,0 +1,50 @@
+          body: JSON.stringify(logsdata),
+        }
+      );
+  
+     // const response = await fetchResponse.json();
+
+     try {
+      //const response = await fetchResponse;
+      const response =  fetchResponse;
+      const contentType = response.headers.get("content-type");
+    
+      if (contentType && contentType.includes("application/json")) {
```

Biome

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)